src/actions/index.js

```diff
@@ -1,10 +1,17 @@
-import images from '../apis/images';
+import { GetRecent, SearchPhoto } from '../apis/flickr';
 import {
-    FETCH_PHOTO
+    GET_RECENT,
+    SEARCH_PHOTO
 } from './types';
 
-export const fetchPhoto = () => async dispatch => {
-    const response = await images.get();
+export const getRecent = () => async dispatch => {
+    const response = await GetRecent.get();
 
-    dispatch({ type: FETCH_PHOTO, payload: response.data.photos.photo});
-};
\ No newline at end of file
+    dispatch({ type: GET_RECENT, payload: response.data.photos.photo});
+};
+
+export const searchPhoto = (term) => async dispatch => {
```

Comment:
This is an interesting pattern. Did you see this in a redux thunk link?

Reply:
Yea, I referenced this link: https://github.com/reduxjs/redux-thunk